src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetExceptionPointersTests.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class GetExceptionPointersTests
+    {
+        [Fact]
+        public void GetExceptionPointers_ReturnsExpected()
+        {
+            Assert.Equal(IntPtr.Zero, Marshal.GetExceptionPointers());
```

Comment:
Is there any additional validation we can/should do for this?

Reply:
I am going to add few additional validation later.